scenarios/test_codegen/test/EventHandler_test.ts

```
@@ -100,4 +100,131 @@ describe("Use Envio test framework to test event handlers", () => {
     //   "The context.addSimpleNft was called after the contract register resolved. Use await or return a promise from the contract register handler to avoid this error."
     // );
   });
+
+  it.only("entity.getOrCreate should create the entity if it doesn't exist", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)